language/German/strings.po

```diff
@@ -3244,11 +3277,11 @@ msgstr "Sound\tOK\n"
 
 #: src/frontend/mame/ui/selmenu.cpp:2924
 msgid "Capture\tUnimplemented\n"
-msgstr ""
+msgstr "Eingabegerät\tNicht implementiert\n"
 
 #: src/frontend/mame/ui/selmenu.cpp:2926
 msgid "Capture\tImperfect\n"
-msgstr ""
+msgstr "Eingabegerät\tFehlerhaft\n"
```

Comment:
This is inconsistent with your use of “Aufzeichnungsgerät” for capture hardware in the unemulated/imperfect features.

Reply:
Thanks for spotting, fixed.